OMCompiler/Makefile.omdev.mingw

```diff
@@ -2,7 +2,7 @@
 CC = gcc
 CXX = g++
 FC = gfortran
-CFLAGS =-g -O2
+CFLAGS =-g -O0
```

Comment:
I guess you don't want to push this in, it should be -g -O2 for the releases.

Reply:
yes this is an error, I will remove it.